evaluator/evaluator.ml

```diff
@@ -53,6 +53,34 @@ end
 
 let setup () = JsooTop.initialize ()
 
+let insertModule name content = 
+  begin 
+    let result = try  
+      let name = Js.to_string name in
+      let content = Js.to_string content in
+      let file_name = ("/static/" ^ (String.capitalize_ascii name) ^ ".re") in 
```

Comment:
I was planning to delegate adding the lang extension to the front-end instead.
We could add a check here and add `.re` as default extension. @thangngoc89

Reply:
Ah. I see. I think we can try `rtop` way of handling language:

https://github.com/facebook/reason/blob/7fa0958f4ef69a4636b4b87f15fb8562ec2c06d2/src/rtop/reason_utop.ml#L31

```ocaml
type top_kind = RTop | UTop
let current_top = ref RTop
```

Because you have to call `reasonSyntax()` and `mlSyntax()` before using `mod_use_file` anyway. But it's up to you. I'll leave this PR to you then